server/middleware/configureEstablishment.js

```diff
@@ -1,17 +1,8 @@
 const { v4: uuid } = require('uuid');
-const {
-  getEstablishmentId,
-  getEstablishmentPersonalisation,
-  getEstablishmentDisplayName,
-} = require('../utils');
+const { getEstablishmentId, getEstablishmentDisplayName } = require('../utils');
 
 const configureEstablishment = () => (req, res, next) => {
-  if (
-    req.session &&
-    (!req.session.id ||
-      !req.session.establishmentId ||
-      req.session.establishmentPersonalisationEnabled === undefined)
-  ) {
+  if (req.session && (!req.session.id || !req.session.establishmentId)) {
     const replaceUrl = /-prisoner-content-hub.*$/g;
 
     const establishmentName = (req.headers?.host || 'wayland')
```

Comment:
I thought we decided to ditch the auto `wayland` behaviour?

Reply:
yes, this is becoming slowly less relevant you can't access the homepage without an establishment now and haven't been able to for a couple of months or so. 

This will be safe to remove entirely going forward